### PR TITLE
Init monorepo project

### PR DESCRIPTION
**Pull request by jiriklic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # hazard-visualisation-tool
-Disaster risk assessment is crucial for effective preparedness and response. 
+
+Disaster risk assessment is crucial for effective preparedness and response.
+
+
```
Comment (Otomisin) [inline README.md:5]: add some comment

File: hazard-visualisation-tool/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "hazard-visualisation-tool"
+version = "0.1.0"
+description = "Disaster risk assessment tool"
+authors = [
+    "Jiri Klic <web@jiriklic.com>"
+]
```
Comment (Otomisin) [inline hazard-visualisation-tool/pyproject.toml:7]: add all contributors

**Review by Otomisin — CHANGES_REQUESTED** [review]
please fix the commetns

File: hazard-visualisation-tool/pyproject.toml (hunk 2 above)
Comment (jiriklic) [reply hazard-visualisation-tool/pyproject.toml:7]: work in progress

**Review by Otomisin — APPROVED** [review]

**Pull request merged**